rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/LocalServers.java

```diff
@@ -33,32 +33,56 @@ public static LocalServer create(LocalServerConfig configuration) {
 
     /** {@link LocalServerConfig} for a {@link LocalServer} with a local storage patch store. */
     public static LocalServerConfig configLocal(String directory) {
+        return configLocal(directory, null);
+    }
+
+    /** {@link LocalServerConfig} for a {@link LocalServer} with a local storage patch store. */
+    public static LocalServerConfig configLocal(String directory, String jettyConf) {
```

Comment:
I created overloaded methods to preserve backward compatibility with other parts of the codebase.